src/libraries/System.Net.HttpListener/ref/System.Net.HttpListener.cs

```diff
@@ -12,6 +12,7 @@ public sealed partial class HttpListener : System.IDisposable
         public HttpListener() { }
         public System.Net.AuthenticationSchemes AuthenticationSchemes { get { throw null; } set { } }
         public System.Net.AuthenticationSchemeSelector? AuthenticationSchemeSelectorDelegate { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
@lewing I assume we wouldn't want to try to take _additional_ `[UnsupportedOSPlatform]` annotations in for 5.0 GA Ask Mode; do you agree?  (I do anticipate we'll take attribute _removals_ like #43387 though).

Reply:
I don't think we should add individual ones at this point.  I think the PR marking the whole assembly is worth considering.

Reply:
yes it will be removed